.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Help with shinyapps.io
+    url: https://forum.posit.co/c/posit-professional-hosted/shinyappsio/24
+    about: For questions about shinyapps.io, visit Posit Community
+  - name: Help with Connect Cloud
+    url: https://forum.posit.co/c/posit-professional-hosted/posit-connect-cloud/67
+    about: For questions about Connect Cloud, visit Posit Community
+  - name: Help with Posit Connect
+    url: https://forum.posit.co/c/posit-professional-hosted/posit-connect/27
+    about: For questions about Posit Connect, visit Posit Community or contact your support rep
```

Comment:
Should we send them straight to the support form? either https://support.posit.co/hc/en-us or https://support.posit.co/hc/en-us/requests/new.

Regarding "support rep", we use "dedicated Posit Customer Success contact" throughout the Connect docs. Maybe "your Posit representative" is sufficient here?

Reply:
sounds good, I'll update